addons/website_slides/controllers/main.py

```diff
@@ -539,7 +539,7 @@ def channel(self, channel, category=None, tag=None, page=1, slide_category=None,
             # display data
             'pager': pager,
             # display upload modal
-            'enable_slide_upload': 'enable_slide_upload' in kw,
+            'enable_slide_upload': (kw.get('enable_slide_upload') if kw.get('enable_slide_upload') != '' else True) if 'enable_slide_upload' in kw else False,
```

Comment:
Can't we simplify to `kw.get('enable_slide_upload') or False` ?

Reply:
Not exactly, but I recognise that this line is a bit ugly, so I simplified it by changing the JS condition associated to it.

(when there is simply "?enable_slide_upload" in the url, the modal should be opened on load, but `kw.get('enable_slide_upload')` returns an empty string, which is considered as false in a conditional statement.